src/vanflow.c

```diff
@@ -1909,6 +1940,8 @@ void vflow_serialize_identity(const vflow_record_t *record, qd_composed_field_t
             snprintf(buffer, IDENTITY_MAX, "%s:%"PRIu64, record->identity.s.source_id, record->identity.record_id);
             qd_compose_insert_string(field, buffer);
         }
+    } else {
```

Comment:
The assert at the top of this function does not belong there.  This will cause crashes in the router when emission is suppressed.